### PR TITLE
[hi] Add content/hi/docs/tutorials/kubernetes-basics/scale/_index.md

### PR DESCRIPTION
**Pull request by verma-kunal**
This PR is a part of ongoing `hi` localization effort here #29353

/language hi

### CODE REVIEW

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM

File: content/hi/docs/tutorials/kubernetes-basics/scale/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: अपने ऐप को स्केल करें
```
Comment (mittalyashu) [inline content/hi/docs/tutorials/kubernetes-basics/scale/_index.md:2]: Can you localise "Application scaling" to Hindi?

Comment (anubha-v-ardhan) [reply content/hi/docs/tutorials/kubernetes-basics/scale/_index.md:2]: `अपने ऐप को स्केल करें` seems the good translation for `Scale Your App`.

**Pull request merged**